courses/migrations/0002_is_visible_deprecated.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from django.contrib.auth.models import User
+from xmodule.modulestore.django import modulestore
+from courses.models import Course
+
+
+def update_mongo_courses(apps, schema_editor):
+    """ Update the mongo "course_visibility" parameter from the SQL value in the FUN database.
+    This is needed because the default value of course_visibility doesn't take account of the show_in_catalog value.
+    """
+
+    mongo_courses = modulestore().get_courses()
+    user_id = User.objects.filter(is_superuser=True).first().id
+    fun_courses = Course.objects.all()
+
+    for mongo_course in mongo_courses:
+        course_id = mongo_course.id
+        print("updating {}".format(course_id))
+
+        try:
+            fun_course = fun_courses.get(key=course_id)
+        except Course.DoesNotExist:
+            pass
+            # if course is not found in fun, we let the default values in mongo
+            print("course not found {}".format(course_id))
+        else:
+            visibility = "none"
+            if fun_course.show_in_catalog:
+                visibility = "both"
+            mongo_course.catalog_visibility = visibility
```

Comment:
Perhaps can we do these last two operations only if necessary:

    if mongo_course.catalog_visibility != visibility:
        mongo_course.catalog_visibility = visibility
        modulestore().update_item(mongo_course, user_id)